README.md

```diff
@@ -5084,46 +5105,58 @@ python3 /opt/NeMo/scripts/nlp_language_modeling/preprocess_data_for_megatron.py
 
-##### 5.16.1.2 Reward Model Training
-<a id="markdown-reward-model-training" name="reward-model-training"></a>
+##### 5.16.1.2 Training a Reward Model
+<a id="markdown-training-a-reward-model" name="training-a-reward-model"></a>
 
-To launch reward model training we first need to start with a pre-trained or fine-tuned nemo checkpoint. Our `training_rm.yaml` file has default configurations for the 2B model but feel free to use any model you like. An example command to begin training is:
+To launch reward model training we first need to start with a pre-trained or fine-tuned NeMo checkpoint. Our `training_rm.yaml` file has default settings for the 2B model but feel free to use any other model (adjusting the config accordingly). An example command to begin training is:
 
 ```bash
 cd /opt/nemo-rlhf \
 && export PYTHONPATH="/opt/nemo-rlhf:${PYTHONPATH}" \
 && python -u examples/nlp/gpt/train_reward_model.py \
     --config-path=examples/nlp/gpt/conf/ \
     --config-name=training_rm \
-    model.pretrained_checkpoint.restore_from_path='model.nemo' \
-    "model.data.data_prefix={train: [${train_output_document}], validation: [${val_output_document}], test: [${test_output_document}]}"
+    exp_manager.explicit_log_dir=/path/to/rm_output_dir \
+    model.pretrained_checkpoint.restore_from_path=/path/to/init_model.nemo \
+    "model.data.data_prefix={train: [/path/to/rm_train], validation: [/path/to/rm_val], test: [/path/to/rm_test]}"
+```
+
+The data files should point to the names of datasets generated as described in the previous section, but without the ".bin" or ".idx" suffix.
+Note that if you are using the command above with your own pre-trained model, you will need to modify `training_rm.yaml` (or the command line) to provide correct values for `tokenizer.model` and `tokenizer.tokenizer_model`.
+You can use `tar tvf /path/to/init_model.nemo` to inspect the model and obtain the name of its tokenizer files: typically, both files are identical and you may thus use the same name for both options, e.g. with
+```bash
+model.tokenizer.model=nemo:2b164b2c1dd74bd691ff90a0db3d39b8_xyz_256k.model \
+model.tokenizer.tokenizer_model=nemo:2b164b2c1dd74bd691ff90a0db3d39b8_xyz_256k.model \
 ```
 
+_Remark: currently, the example training script does not automatically run evaluation on the provided test set. This may change in a future release._
+
 ##### 5.16.1.3 Reward Model Evaluation
 <a id="markdown-reward-model-evaluation" name="reward-model-evaluation"></a>
 
-To learn how to serve the reward model for evaluation, see the section "Launching the Reward Model inference server" below.
+Once trained, a reward model may be served for evaluation purpose, as described in the section "Launching the Reward Model Inference Server" below.
+This can also useful to compute the mean / std of reward predictions before doing PPO training, to be able to normalize them: documentation and scripts to perform such normalization will be provided soon.
```

Comment:
I think we need to move over (and properly test) some of our scripts inside the main repository (as examples) before we can document reward normalization.